src/main/daml/Daml/Finance/Interface/Settlement/Instruction.daml

```diff
@@ -45,14 +45,14 @@ interface Instruction where
         -- ^ The party retrieving the view.
```

Comment:
Was this file reverted to a previous version unintentionally?

Reply:
yes, it seems like that. but I made the changes (of removing `templateTypeRepToText`) from my old branch, so should be fine now.